### PR TITLE
Add logging formatter

### PR DESCRIPTION
**Pull request by garuppel**
Add formatter for capitalize log messages. #537

### CODE REVIEW

**Review by lebogg — COMMENTED** [review]
Seems to be good. Just some minor questions.

File: cmd/engine/engine.go [hunk 1]
```diff
@@ -26,6 +26,7 @@
 package main
 
 import (
+	"clouditor.io/clouditor/logging/formatter"
```
[the remark this answers is not in the document]
Comment (garuppel) [reply cmd/engine/engine.go:29]: I don't know why Intellij is doing it like that. I put the two blocks together.

File: cmd/engine/engine_test.go [hunk 2]
```diff
@@ -9,6 +11,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func init() {
```
[the remark this answers is not in the document]
Comment (garuppel) [reply cmd/engine/engine_test.go:14]: The purpose is to use the CapitalizeFormatter in the test.

[the remark this answers is not in the document]
Comment (oxisto) [reply cmd/engine/engine_test.go:14]: Yes. Test_Main is probably the better way to initialise it.

[the remark this answers is not in the document]
Comment (oxisto) [reply cmd/engine/engine_test.go:14]: hmm although this would be a major drawback, that we would need a lot of TestMains for test files where we have none. The only issue that remains would be if this somehow overrides a log that exists in the same package already?

[the remark this answers is not in the document]
Comment (garuppel) [reply cmd/engine/engine_test.go]: I deleted all init() methods from the test files.

**Review by oxisto — APPROVED** [review]
